src/custom/hooks/useFetchProfile.ts

```diff
@@ -3,22 +3,40 @@ import { useEffect, useState } from 'react'
 import { getProfileData } from 'api/gnosisProtocol'
 import { ProfileData } from 'api/gnosisProtocol/api'
 
-export default function useFetchProfile() {
+type FetchProfileState = {
+  profileData: ProfileData | null
+  //error: string
+  isLoading: boolean
+}
+
+const emptyState: FetchProfileState = {
+  profileData: null,
+  //error: '',
```

Comment:
Dead code?

Reply:
I left that commented to hint Henry to put there the error in https://github.com/gnosis/cowswap/pull/1745